index.html

```diff
@@ -0,0 +1,246 @@
+<!doctype html>
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+            integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        <script src="https://kit.fontawesome.com/8c36836212.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="style.css">
+        <title>Nakuru Snooker Tour</title>
+        <link rel="icon" href="./imgs/billiard-ball.png" type="image/png">
+    </head>
+
+    <body>
+        <header>
+            <nav class="navbar navbar-expand-md d-none d-print-flex d-sm-none d-md-flex d-lg-flex d-xl-flex d-xxl-flex py-3 nav-dark"
+                data-bs-theme="dark">
+                <div class="container">
+                    <div id="navcol-5" class="collapse navbar-collapse show nav-collapse">
+                        <ul class="navbar-nav ms-auto">
+                            <li class="nav-item"><a class="nav-link active white-link" href="facebook.com"
+                                    target="_blank" rel="noopener"><i class="fab fa-facebook-f"></i></a></li>
+                            <li class="nav-item"><a class="nav-link active white-link" href="twitter.com"
+                                    target="_blank" rel="noopener"><i class="fab fa-twitter"></i></a></li>
+                            <li class="nav-item"><a class="nav-link active white-link" href="#">English</a></li>
+                            <li class="nav-item"><a class="nav-link white-link" href="#">My Page</a></li>
+                            <li class="nav-item"><a class="nav-link white-link" href="#">Logout</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </nav>
```

Comment:
- [ ] Please add a bottom border shadow of the navbar on both pages to match the given design
- [ ] Also in the mobile version, the navbar's background should be transparent. Please fix it.

yours|given
---|---
![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/91486791/2109102d-c0f3-47ba-959f-504421e24658)|![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/91486791/2170c2c1-a0df-4c8b-8168-6e9919fc4ca8)
![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/91486791/ebca547b-448b-47cf-8809-8bf6ebb5501d)|![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/91486791/414c35e5-92fb-44be-915a-2fd5808d1713)